apis/mattermost/v1beta1/mattermost_utils.go

```diff
@@ -35,7 +35,7 @@ const (
 
 	// ClusterResourceLabel is the label applied to a given Mattermost
 	// as well as all other resources created to support it.
-	ClusterResourceLabel = "v1beta1.installation.mattermost.com/resource"
+	ClusterResourceLabel = "v1alpha1.mattermost.com/resource"
```

Comment:
Should this be `v1beta1`?

Reply:
Decided to leave it the same as in ClusterInstallation for now, so that it can work with current Network Policies setup. The labels change will come in the next PR.